packages/loader/container-loader/src/containerContext.ts

```diff
@@ -324,13 +324,10 @@ export class ContainerContext implements IContainerContext {
 
     private async instantiateRuntime(existing: boolean) {
         const runtimeFactory = await this.getRuntimeFactory();
-        await ReportIfTooLong(
-            this.taggedLogger,
-            "instantiateRuntime",
-            async () => {
-                this._runtime = await runtimeFactory.instantiateRuntime(this, existing);
-                return {};
-            });
+        this._runtime = await PerformanceEvent.timedExecAsync(
```

Comment:
this change seems fine, but i wonder if more a sustainable long term approach would be to have enough observability points that a consumer could measure this directly if they care. nothing specifically to change here, just something to consider longer term.

Reply:
@anthony-murphy , I agree in the general case. For this specific case, we're already doing a "ReportIfTooLong", so switching it reporting it all the time doesn't seem particularly problematic. And we're already intertwining FF telemetry with container telemetry anyway.

So yes, duly noted in general; but for this specific instance, going ahead and resolving this thread (and note there's a similar https://github.com/microsoft/FluidFramework/pull/13429 as well)